core/domain/feedback_services_test.py

```diff
@@ -431,21 +378,12 @@ def test_get_total_open_threads_for_multiple_explorations(self):
             'exploration', self.EXP_ID_2, False)
         self.assertEqual(1, len(threads_exp_2))
 
-        def _close_thread(thread_id):
-            """Closes the thread corresponding to its thread id by updating its
-            status.
-            """
-            thread = feedback_models.GeneralFeedbackThreadModel.get_by_id(
-                thread_id)
-            thread.status = feedback_models.STATUS_CHOICES_FIXED
-            thread.update_timestamps()
-            thread.put()
-
-        _close_thread(threads_exp_1[0].id)
+        feedback_services.create_message(
+            threads_exp_1[0].id, self.user_id,
+            feedback_models.STATUS_CHOICES_FIXED, None, 'not used here')
```

Comment:
Maybe say 'feedback message not used here' or similar (took me a while to realize it was referring to that particular argument rather than the overall message object).

Reply:
Done.